src/core/namespace.c

```diff
@@ -1173,7 +1173,7 @@ static int apply_mount(
                 return mount_run(m);
 
         case MOUNT_IMAGES:
-                return mount_images(m);
+                return mount_image(m, NULL);
```

Comment:
hmm, the function only has only one argument further up? rebase mistake? /me is confused

Reply:
yep, refactor mistake - I've split the commit that moves/renames from the "new feature commit", and this fell through, fixed now